From 9959e2e9017a64a4a336fa481a743d2529d109ad Mon Sep 17 00:00:00 2001
From: hasufell <hasufell@gentoo.org>
Date: Tue, 20 Oct 2015 16:38:33 +0200
Subject: [PATCH] Fix compilation with system angelscript

Upstream names the shared library angelscript_s.
In addition, add more common PATH_SUFFIXES.
---
 cmake/FindAngelscript.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/FindAngelscript.cmake b/cmake/FindAngelscript.cmake
index 09020c3..8cfa56b 100644
--- a/cmake/FindAngelscript.cmake
+++ b/cmake/FindAngelscript.cmake
@@ -14,11 +14,11 @@ FIND_PATH(Angelscript_INCLUDE_DIRS angelscript.h
     )
 
 FIND_LIBRARY(Angelscript_LIBRARY
-    NAMES angelscript
+    NAMES angelscript angelscript_s
     PATHS
     /usr/local
     /usr
-    PATH_SUFFIXES lib
+    PATH_SUFFIXES lib lib64 lib32
     )
 
 # handle the QUIETLY and REQUIRED arguments and set ANGELSCRIPT_FOUND to TRUE if
-- 
2.6.1

